README.md

```diff
@@ -239,6 +239,10 @@ alertmanager:
   # expiresafter: "" if set to a non-zero value, alert expires after that time in seconds (default: 0)
   # extralabels: "" # comma separated list of labels composed of a ':' separated name and value that is added to the Alerts. Example: my_label_1:my_value_1, my_label_1:my_value_2
   # extraannotations: "" # comma separated list of annotations composed of a ':' separated name and value that is added to the Alerts. Example: my_annotation_1:my_value_1, my_annotation_1:my_value_2
+  # dropeventdefaultpriority: "" # default priority of dropped events, values are emergency|alert|critical|error|warning|notice|informational|debug (default: "critical")
+  # dropeventthresholdlist: # List of priority re-evaluation thresholds of dropped events. Default: "10000:critical, 1000:critical, 100:critical, 10:warning, 1:warning"
```

Comment:
Should be comma separated values to be able to be set with an env var, like the others